echo/echo_alias.go

```diff
@@ -0,0 +1,8 @@
+package echo
+
+import "github.com/labstack/echo/v4"
+
+// Context is an alias/composition of echo.Context that allows generating a mock of echo.Context
+type Context interface {
```

Comment:
maybe move it to `test/echo`?

Reply:
the `echo/` package only contains tests, maybe just rename that to `echo_test/`.

Reply:
The strict open-API interfaces seem to remove the need for mocking of echo.Context completely.